_episodes_rmd/30-dplyr.Rmd

```diff
@@ -848,6 +848,95 @@ so every replicate has the same composition.
 
 
 
+
+## Joining tables
+
+In many real life situations, data are spread across multiple tables.
+Usually this occurs because different types of information are collected 
+from different sources. 
+
+It may be desirable for some analyses to combine data from two or more 
+tables into a single data frame based on acolumn that would be common 
+to all the tables.
+
+The `dplyr` package provides a set of join functions for combining 
+two data frames based on matches within specified columns.
+
+For further reading, please refer to the chapter about [table
+joins](https://r4ds.had.co.nz/relational-data.html#understanding-joins)
+in @r4ds:2017.
+
+The [Data Transformation Cheat
+Sheet](https://github.com/rstudio/cheatsheets/raw/master/data-transformation.pdf)
+also provides a short overview on table joins.
+
+We are going to illustrate join using a first table, `rna_mini`, which 
+is just a subset of the original `rna` table.
+
+```{r}
+load(file = "data/rna_mini.rda")
```

Comment:
Could you please use the long `rna` data that was used throughout the course for the joining example instead of having to load a new one.